SETUP/ci/check_require_login.php

```diff
@@ -26,9 +26,7 @@
     // API
     "api/index.php",
     // FAQs
-    "faq/default.php",
```

Comment:
Removed this as it redirected to the now defunct `faq_central.php`. This means that the `faq/` URL will now return some PHP error.